client/README.md

```diff
@@ -0,0 +1,15 @@
+# Bootz Client Emulator
+
+The code located in this directory is intended to emulator a typical Bootz client. Where appropriate, some device-specific functions such as downloading and image or applying a config are mocked out and are simply logged.
```

Comment:
shouldn't they be aligned?

Reply:
For the protocol version, yes -- but let's say you want to have a v2 of some of the client library which changes the API without changing the proto/service that is being referenced, doing this today would be difficult since you need to have the same tag for both the repo.

Reply:
as we spoke we will align the go mod while also keeping the proto version